torchaudio/models/_wavernn.py

```diff
@@ -205,54 +205,54 @@ class _WaveRNN(nn.Module):
         sample_rate: the rate of audio dimensions (samples per second)
         hop_length: the number of samples between the starts of consecutive frames
         n_res_block: the number of ResBlock in stack (default=10)
         n_rnn: the dimension of RNN layer (default=512)
         n_fc: the dimension of fully connected layer (default=512)
         kernel_size: the number of kernel size in the first Conv1d layer (default=5)
         n_freq: the number of bins in a spectrogram (default=128)
-        n_hidden: the number of hidden dimensions (default=128)
-        n_output: the number of output dimensions (default=128)
-        mode: the mode of waveform in ['waveform', 'mol'] (default='waveform')
+        n_hidden_resblock: the number of hidden dimensions of resblock (default=128)
+        n_output_melresnet: the number of output dimensions of melresnet (default=128)
+        loss: the type of loss in ['crossentropy', 'mol'] (default='crossentropy')
 
     Example
-        >>> wavernn = _waveRNN(upsample_scales=[5,5,8], n_bits=9, sample_rate=24000, hop_length=200)
+        >>> wavernn = _waveRNN(upsample_scales=[5,5,8], n_classes=512, sample_rate=24000, hop_length=200)
         >>> waveform, sample_rate = torchaudio.load(file)
         >>> # waveform shape: (n_batch, n_channel, (n_time - kernel_size + 1) * hop_length)
         >>> specgram = MelSpectrogram(sample_rate)(waveform)  # shape: (n_batch, n_channel, n_freq, n_time)
         >>> output = wavernn(waveform, specgram)
-        >>> # output shape: (n_batch, n_channel, (n_time - kernel_size + 1) * hop_length, 2 ** n_bits)
+        >>> # output shape: (n_batch, n_channel, (n_time - kernel_size + 1) * hop_length, n_classes)
     """
 
     def __init__(self,
                  upsample_scales: List[int],
-                 n_bits: int,
+                 n_classes: int,
                  sample_rate: int,
                  hop_length: int,
                  n_res_block: int = 10,
                  n_rnn: int = 512,
                  n_fc: int = 512,
                  kernel_size: int = 5,
                  n_freq: int = 128,
-                 n_hidden: int = 128,
-                 n_output: int = 128,
-                 mode: str = 'waveform') -> None:
+                 n_hidden_resblock: int = 128,
+                 n_output_melresnet: int = 128,
+                 loss: str = 'crossentropy') -> None:
         super().__init__()
 
-        self.mode = mode
+        self.loss = loss
         self.kernel_size = kernel_size
 
-        if self.mode == 'waveform':
-            self.n_classes = 2 ** n_bits
-        elif self.mode == 'mol':
+        if self.loss == 'crossentropy':
+            self.n_classes = n_classes
+        elif self.loss == 'mol':
             self.n_classes = 30
         else:
-            raise ValueError(f"Expected mode: `waveform` or `mol`, but found {self.mode}")
+            raise ValueError(f"Expected loss: `crossentropy` or `mol`, but found {self.loss}")
```

Comment:
We discussed offline that we would remove the "loss/mode" parameter for WaveRNN. Let's do it as part of this PR.

Reply:
Yes, I have removed it. I also removed the `sample_rate` parameter since it is not needed.